pom.xml

```diff
@@ -29,7 +29,7 @@
 		<gitHubRepo>jenkinsci/scriptler-plugin</gitHubRepo>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <maven.javadoc.skip>true</maven.javadoc.skip>
-		<jenkins.version>2.387.3</jenkins.version>
+		<jenkins.version>2.426</jenkins.version>
```

Comment:
Using 2.426 because 2.426.1 is not out yet, but this can be changed to 2.426.1 LTS when it is available.

Reply:
Thanks. I have changed this to the LTS release, given that it's out now.